src/source/mod.rs

```diff
@@ -8,7 +8,7 @@ mod statsd;
 pub use self::file::{FileServer, FileServerConfig};
 pub use self::flush::FlushTimer;
 pub use self::graphite::{Graphite, GraphiteConfig};
-pub use self::internal::{Internal, InternalConfig};
+pub use self::internal::{Internal, InternalConfig, report_telemetry};
```

Comment:
@pulltab it is defined in internal.rs, here: https://github.com/postmates/cernan/blob/master/src/source/internal.rs#L57